maven-plugin/plugin-core/src/main/java/org/jvnet/jaxb/maven/AbstractXJC2Mojo.java

```diff
@@ -446,6 +446,19 @@ public boolean getDisableSystemIdResolution() {
     public void setDisableSystemIdResolution(boolean disableSystemIdResolution) {
         this.disableSystemIdResolution = disableSystemIdResolution;
     }
+    /**
+     * If 'true', the fix for issue #19 will be applied.
+     */
+    @Parameter(defaultValue = "false", property = "maven.xjc2.relativeCatalogResolution")
+    private boolean relativeCatalogResolution;
+
+    public boolean getRelativeCatalogResolution() {
+        return relativeCatalogResolution;
```

Comment:
Should be `is`, no?

Reply:
There is already some boolean parameter with "get". It has no direct impact on resolution for maven / mojo.